src/cascade/executor/dismod_runner.py

```diff
@@ -191,14 +193,14 @@ def _async_run_and_watch(command, single_use_machine, poll_time):
     try:
         CODELOG.info(f"Forking to {command}")
         sub_process = yield from asyncio.subprocess.create_subprocess_exec(
-            *command, stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE, preexec_fn=pre_execution_function
+            *command, stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE, preexec_fn=pre_execution_function,
+            loop=loop
```

Comment:
I am making it explicit because I'm creating a new loop each go around. Though I am setting it as the default event loop on the process. Actually, now that I think about it I can probably setup a new loop when I enter the child process, register it with the process and be done. That's cleaner I think... I need to invest a little bit more time in understanding what's going on here. Despite the complexity of this I still think this is the cleanest way to track the subprocess output though if it get's more tangled I'm going to have to reevaluate.